contracts/staking/GoodFundManager.sol

```diff
@@ -91,14 +89,6 @@ contract GoodFundManager is DAOContract {
         // from the staking contract to the
         // reserve contract
         uint256 cDAIinterestEarned,
-        // How much interest has been donated
-        // according to the given donation
-        // ratio which determined in the
-        // staking contract
-        uint256 cDAIinterestDonated,
-        // The number of tokens that have been minted
-        // by the reserve to the staking contract
-        uint256 gdInterest,
```

Comment:
i dont understand why you removed everything, we discussed we will handle it in the relevant issue once we have decission what to do with donation